cmd/config/internal/commands/cmdset.go

```diff
@@ -30,6 +30,8 @@ func NewSetRunner(parent string) *SetRunner {
 	}
 	fixDocs(parent, c)
 	r.Command = c
+	c.Flags().StringVar(&r.Set.Value, "value", "",
```

Comment:
This should be a string slice for the setting a list case

Reply:
that would be cleaner, modified